apps/rates-service/package.json

```diff
@@ -20,11 +20,7 @@
     "dev": "cross-env TS_NODE_PROJECT=../../tsconfig.dev.json node --inspect=9231 -r ts-node/register -r ./src/main.ts",
     "dev:live": "cross-env CONFIG_NAME=devlive yarn dev",
     "prod:test": "cross-env CONFIG_NAME=prodtest yarn dev",
-    "predeploy": "ts-node --skip-project predeploy",
-    "deploy": "run-s deploy:src deploy:cron",
-    "deploy:src": "gcloud app deploy",
-    "deploy:cron": "gcloud app deploy cron.yaml",
-    "postdeploy": "ts-node --skip-project ../../tools/removeOldAppVersions",
+    "deploy": "ts-node --skip-project deploy",
```

Comment:
All the complexity is now moved into the `deploy.ts` file, where it is much easier to manage than out here in package scripts file!